FirebaseCore/Tests/Unit/FIRHeartbeatInfoTest.m

```diff
@@ -29,7 +29,7 @@ @implementation FIRHeartbeatInfoTest
 - (void)setUp {
   NSString *const kHeartbeatStorageFile = @"HEARTBEAT_INFO_STORAGE";
   self.dataStorage = [[GULHeartbeatDateStorage alloc] initWithFileName:kHeartbeatStorageFile];
-  NSDate *pastTime = [NSDate dateWithTimeIntervalSinceNow:-996400];
+  NSDate *pastTime = [NSDate dateWithTimeIntervalSinceNow:-99640];
```

Comment:
I would suggest using `NSCalendar` and `NSComponents` to calculate the date in a more readable way, e.g. 
```
NSDateComponents *componentsToAdd = [[NSDateComponents alloc] init];
componentsToAdd.day = -1;

NSDate *dateYesterday = [[NSCalendar currentCalendar] dateByAddingComponents:componentsToAdd toDate:[NSDate date] options:0];
```

It may look a bit overkill, but I think it is worth it taking into account the context of the issue.